lib/tasks/content_change_email.rake

```diff
@@ -1,8 +1,18 @@
 namespace :report do
-  desc "Produce a report on sent, pending and failed emails for a given content change"
-  task :content_change_email_status_count, [:id] => :environment do |_t, args|
-    content_change = ContentChange.find(args[:id])
-    Reports::ContentChangeEmailStatusCount.call(content_change)
+  desc "Produce a report on sent, pending and failed emails for given content change id or ids"
+  task content_change_email_status_count: :environment do |_t, args|
+    ids = args.extras
+    if ids.present?
+      ids.each do |id|
+        content_change = ContentChange.find(id)
```

Comment:
Perhaps we should do `ContentChange.where(id: id)` and then iterate over the `ContentChanges` to avoid unnecessary queries.